src/lib/menu/menu-directive.ts

```diff
@@ -141,6 +141,15 @@ export class MatMenu implements AfterContentInit, MatMenuPanel, OnDestroy {
     }
   }
 
+  /**
+   * This method takes classes set on the host mat-menu element and applies them on the
+   * menu template that displays in the overlay container.  Otherwise, it's difficult
+   * to style the containing menu from outside the component.
+   * @deprecated Use `panelClass` instead.
+   * @param classes list of class names
+   */
+  @Input() classList = this.panelClass;
```

Comment:
This needs a getter/setter that delegates to `panelClass`; this will only set the initial value